packages/@jsii/dotnet-analyzers/src/NuGet.config

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do these have a NuGet.config? Are these published to NuGet?

Reply:
The NuGet.config file was used to configure a "local source" so local builds work before publishing. Since now in a single solution, this is no longer needed!